packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -67,21 +67,6 @@ export class AnchorMenuItem extends AnchorBase {
         return true;
     }
```

Comment:
There is a workaround cited at the top of this file linking to an existing fast issue, can we push that one forward? i.e. make a legacy branch pr and master branch PR?
https://github.com/microsoft/fast/issues/5415

Reply:
It seems like they want to address that by introducing a new anchor menu item element into FAST. Maybe that's a simple as moving our implementation into FAST, but there's some uncertainty there, and I'm not sure we want to prioritize it. We shouldn't make this PR wait on that.

Reply:
> Maybe that's a simple as moving our implementation into FAST, but there's some uncertainty there, and I'm not sure we want to prioritize it. We shouldn't make this PR wait on that.

This comment was resolved but it's not clear how are we capturing following up on that?

> We shouldn't make this PR wait on that.

If a comment is closed with that resolution make sure it's clear how we track following up on that.

Reply:
Filed tech debt issue: https://github.com/ni/nimble/issues/1740